browser/admin/admintemplate.html

```diff
@@ -7,7 +7,7 @@
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
     <meta name="description" content=""/>
     <meta name="author" content=""/>
-    <link rel="localizations" href="%SERVICE_ROOT%/browser/%VERSION%/l10n/admin-localizations.json" type="application/vnd.oftn.l10n+json"/>
```

Comment:
`type` is something you  want to delete?

Reply:
This change looks fine to me, admin-localizations.json was an old concept. localizations.json is basically the same, but with absolute URLS.

Reply:
Mistake :)

Reply:
then code is good for me